domain_access/src/DomainAccessManager.php

```diff
@@ -97,6 +95,7 @@ public static function getDefaultValue(FieldableEntityInterface $entity, FieldDe
     switch ($entity->getEntityType()) {
       case 'user':
       case 'node':
+        // @TODO $this is no accessible in a static context.
```

Comment:
No, I get it now. It's a static method because the Form system might call it at anytime, blindly.

We should refactor in a follow-up.